glotaran/utils/io.py

```diff
@@ -186,3 +186,41 @@ def relative_posix_path(source_path: StrOrPath, base_path: StrOrPath | None = No
         except ValueError:
             pass
     return Path(source_path).as_posix()
+
+
+def safe_parameters_fillna(df, column_name, fill_value):
+    """Ensure that columns exist in order to replace empty strings with +/-np.inf values.
+
+    Parameters
+    ----------
+    df : pd.DataFrame
+        DataFrame from which specific column values will be replaced
+    column_name : str
+        Name of column from DataFrame
+    fill_value : str
+        Values to be replaced in column
+    """
+    if column_name in df.columns:
+        df[column_name].fillna(fill_value, inplace=True)
+
+
+def safe_parameters_replace(df, column_name, to_be_replaced_values, replace_value):
```

Comment:
This function needs tests.

Reply:
Added by @patrickhaetti in 7233dbce5540b769a520b8e648b3be669e5deafb